src/main/kotlin/com/cognifide/gradle/aem/common/file/transfer/FileTransfer.kt

```diff
@@ -15,6 +15,11 @@ interface FileTransfer {
      */
     val enabled: Boolean
 
+    /**
+     * Determines if operations using this transfer could be done in parallel.
+     */
+    val parallelable: Boolean
+
```

Comment:
it matters because as of Gradle 6 it is not possible to resolve files from different configurations in parallel